controllers/machinehealthcheck_controller.go

```diff
@@ -267,7 +272,8 @@ func (r *MachineHealthCheckReconciler) reconcile(ctx context.Context, logger log
 	}
 	for _, t := range healthy {
 		if err := t.patchHelper.Patch(ctx, t.Machine); err != nil {
-			return ctrl.Result{}, errors.Wrapf(err, "Failed to patch healthy machine status for machine %q", t.Machine.Name)
+			errList = append(errList, errors.Wrapf(err, "Failed to patch healthy machine status for machine %q", t.Machine.Name))
+			continue
```

Comment:
Is this change not related to this PR?

Reply:
It's related since it's where I noticed the problem of the unused `errList` and realized I didn't intentionally mean to change the logic of MHC to not try to patch each machine and instead return early after the first error.